src/routes/foundations/Typography.tsx

```diff
@@ -1,9 +1,125 @@
 
 export default function FoundationsTypographyPage() {
   return (
       <ComponentContent>
         <h1>Typography</h1>
+         <GoADivider mt="2xl" mb="2xl"></GoADivider>
+
+        <h2>Importance of visual hierarchy</h2>
+        <p>Defining a visual hierarchy in text design is essential for shaping the visual narrative, enhancing legibility, and conveying the importance of content by arranging it at different levels.</p>
+        <ul>
+          <li><strong>Visual hierarchy</strong>: Our typography scale helps in creating a clear structure by distinguishing different levels of information, guiding users through the content based on importance and flow.</li>
+          <li><strong>Emphasis</strong>: Using font variations like size, weight and style, we can draw attention to key elements or actions and prioritize certain information over others.</li>
+          <li><strong>Purpose</strong>: Our typeface supports the message by aligning the text's style with the intended meaning of the content.</li>
+          <li><strong>Legibility</strong>: Ensuring that the text is easy to read across all devices and context is crucial, and our typography plays a key role by optimizing font size, spacing, and contrast.</li>
+        </ul>
+
+        <h2>Typefaces</h2>
+        <h3>Acumin Pro SemiCondensed</h3>
+        <p>We use the <a href="https://fonts.adobe.com/fonts/acumin" target="_blank">Acumin Pro SemiCondensed</a> typeface for all the web interfaces in the Government of Alberta (GoA). Follow our <Link to="/get-started/developers/setup">setup guide</Link>, and the necessary fonts are automatically included.</p>
+
+        <h3>Roboto</h3>
+        <p>We use the open-sourced monospaced typeface, <a href="https://fonts.google.com/specimen/Roboto+Mono?subset=cyrillic-ext&selection.family=Alegreya+Sans+SC:ital,wght@0,100;0,300;0,400;0,500;0,700;0,800;0,900;1,100;1,300;1,400;1,500;1,700;1,800;1,900|Alegreya+Sans:ital,wght@0,100;0,300;0,400;0,500;0,700;0,800;0,900;1,100;1,300;1,400;1,500;1,700;1,800;1,900|Arimo:ital,wght@0,400;0,700;1,400;1,700|Arsenal:ital,wght@0,400;0,700;1,400;1,700|Comfortaa|Cormorant+Infant:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700|Cousine:ital,wght@0,400;0,700;1,400;1,700|Cuprum|Didact+Gothic|Fira+Mono:wght@400;500;700|Fira+Sans+Condensed|Fira+Sans+Extra+Condensed:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;1,100;1,200;1,300;1,400;1,500;1,600&preview.text=te&preview.text_type=custom" target="_blank">Roboto Mono</a> for numbers in places where comparing numbers is important. This enhances readability and alignment when displaying numerical data, making it easier to compare figures accurately.</p>
+        
+        <h2>Text styles</h2>
+        <GoABlock mt="xl" mb="xl">
+          <div style={{ font: getCssVarValue(`--$goa-typography-heading-xl`) }}>XLarge heading - Perspiciatis unde omnis iste natus error sit.</div>
```

Comment:
remove the `$`

Reply:
Removed them.

Reply:
Is this what sparked that flagged message you mentioned @chrisolsen?

Reply:
For the future, the use of the `getCssVarValue` is not needed. Just using `font:  "var(--goa-typo...)"` should work.